README.md

```diff
@@ -55,234 +20,121 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
+- [❓ FAQ](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+# 📖 Capstone 1 The LOTR Lovers-Conference <a name="about-project"></a>
+
 
-**[your_project__name]** is a...
 
-## 🛠 Built With <a name="built-with"></a>
+**[Capstone 1 The LOTR Lovers-Conference is my second project and I prepared a conference project]**
 
+## 🛠 Built With<a name="built-with"></a>
+ <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+    <li>Git</li>
+    <li>GitHub</li>
+  </ul>
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
 
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+
+- **[Creating linters and using them]**
+- **[Creating JS, html and css files]**
+- **[Opening a pull request]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+URL:https://otmantr.github.io/Capstone-1.-The-LOTR-Lovers-Conference/
```

Comment:
- [ ] As required of you in the project requirements, you are required to deploy your project to be able to view it online. Kindly deploy your project to either GitHub pages or Netlify so that it is accessible online.

![image](https://user-images.githubusercontent.com/86318284/212120524-d4009665-8fa6-4ef5-b7e2-17a24741d12c.png)

Reply:
- [ ] @otmanTR great work implementing the changes requested however you forgot to update the link in your readme kindly do so, currently, it leads to an empty page
![image](https://user-images.githubusercontent.com/61408860/212462915-21de4118-a2b7-4f50-816d-adf610e50680.png)